recipes/npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
remove this file with a new commit. After the commit, update your `.gitignore` to ignore `npm-debug.log`